mdpow/tests/test_dihedral.py

```diff
@@ -37,25 +37,25 @@ def teardown(self):
         self.tmpdir.dissolve()
 
     def test_dataframe(self):
-        dh1 = self.Ens.select_atoms('name C4 or name C17 or name S2 or name N3')
+        dh1 = self.Ens.select_atoms('name C4', 'name C17', 'name S2', 'name N3')
         dh_run = DihedralAnalysis([dh1]).run(start=0, stop=4, step=1)
 
         results = dh_run.results
 
-        assert results['selection'][0] == 'S2-N3-C4-C17'
+        assert results['selection'][0] == 'C4-C17-S2-N3'
```

Comment:
revert changes, will be fixed in PR #218

Reply:
selection order consistent with original selection string

Reply:
My point was that each PR should address _separate_ issues and there should not be any duplicated code changes. PRs should be "orthogonal" to each other and not contain each other's work. In this case it does not seem to matter because I was able to merge PR #218 , which properly fixes the problem, into _this_ PR without problem but this is not always straightforward. And conceptually it's messy: It's best when one PR fixes one problem.